### DISCUSSION TITLE
I don't want to show Exp name:xxxxxxx

### DISCUSSION THREAD

**Discussion by khh0106** [open-mmlab/mmclassification#1492] [Q&A] [answered]
How can I make the message below invisible?
![image](https://user-images.githubusercontent.com/79922343/232193578-6d4662de-4deb-4987-b5e6-401d207168a8.png)

**Comment by khh0106** [comment]
logger=dict(type='LoggerHook', interval=10, interval_exp_name=10000000)
I change interval_exp_name but occur again

**Reply by Ezra-Yu** [reply, marked as answer]
here is the doc of [LoggerHook](https://mmengine.readthedocs.io/en/latest/api/generated/mmengine.hooks.LoggerHook.html#loggerhook). There is no param of `interval_exp_name`. If you want to modify the exp name, you can rename it by the configuration the Runnr refer to the [doc](https://mmengine.readthedocs.io/en/latest/api/generated/mmengine.runner.Runner.html#runner).

add the `experiment_name` in cfg:
```
model = xxx
train_dataloader = xxxx
.....
experiment_name = 'Your_EXP_NAME'
```

**Reply by khh0106** [reply]
Thank you for your information
